website/content/docs/operators/deployment/schedulers/dcos.md

```diff
@@ -98,11 +99,8 @@ heron.marathon.scheduler.uri: "<core.dcos_url>/service/marathon"
 # Invoke the IScheduler as a library directly
 heron.scheduler.is.service:                  False
 
-# location of the core package
-heron.package.core.uri:                      https://github.com/twitter/heron/releases/download/0.14.7/heron-core-0.14.7-ubuntu.tar.gz
-
-# docker repo for executor
-heron.executor.docker.image: 'ndustrialio/heron-executor:jre8'
+# docker repo for heron with core packages installed
+heron.executor.docker.image: 'streamlio/heron:latest-ubunut14.04'
```

Comment:
We are planning on publishing images for different operating systems so we came up with the convention heron:<heron-version>-<os-version>. So would publish a release image like this streamlio/heron:0.14.8-ubuntu14.04. What do you think?